srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -0,0 +1,104 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.validation.Validator;
+
+import gov.va.med.srcalc.domain.model.RiskModel;
+import gov.va.med.srcalc.domain.model.Specialty;
+
+public class EditRiskModel implements Comparable<EditRiskModel>{
+
+	private final RiskModel riskModel;
```

Comment:
By convention, all of our objects' fields are prefixed with `f`. So this would be `fRiskModel`.